src/utils/helpers/currencies.ts

```diff
@@ -1,9 +1,20 @@
 import { CurrencyExt } from '@interlay/interbtc-api';
+import { MonetaryAmount } from '@interlay/monetary-js';
 
 // Squid query by currency ticker for native assets and by id for foreign assets.
 // We need to differentiate because those are handled differently on squid side.
 // TODO: Need to refactor when we want to support lend tokens as collateral for vaults.
 const getCurrencyEqualityCondition = (currency: CurrencyExt): string =>
   'foreignAsset' in currency ? `asset_eq: ${currency.foreignAsset.id}` : `token_eq: ${currency.ticker}`;
 
-export { getCurrencyEqualityCondition };
+const pickSmallerAmount = (
+  amount0: MonetaryAmount<CurrencyExt>,
+  amount1: MonetaryAmount<CurrencyExt>
+): MonetaryAmount<CurrencyExt> => {
+  if (amount0.lte(amount1)) {
```

Comment:
Just make it a ternary

Reply:
I think the current syntax is easier to read than ternary actually - not a strong opinion though, can change if you feel strongly about this.)